.github/workflows/integration_tests.yml

```diff
@@ -41,21 +41,25 @@ jobs:
         dbt run-operation dbt_external_tables.stage_external_sources --vars 'ext_full_refresh: true' --target ${{ matrix.data-platform }}
         dbt run-operation dbt_external_tables.stage_external_sources --target ${{ matrix.data-platform }}
         dbt test --target ${{ matrix.data-platform }}
-      env: 
-        REDSHIFT_TEST_HOST: ${{ secrets.REDSHIFT_TEST_HOST }}
-        REDSHIFT_TEST_USER: ${{ secrets.REDSHIFT_TEST_USER }}
-        REDSHIFT_TEST_PASS: ${{ secrets.REDSHIFT_TEST_PASS }}
-        REDSHIFT_TEST_PORT: ${{ secrets.REDSHIFT_TEST_PORT }}
-        REDSHIFT_TEST_DBNAME: ${{ secrets.REDSHIFT_TEST_DBNAME }}
+      env:
+        # redshift
+        REDSHIFT_HOST: ${{ secrets.REDSHIFT_TEST_HOST }}
+        REDSHIFT_USER: ${{ secrets.REDSHIFT_TEST_USER }}
+        DBT_ENV_SECRET_REDSHIFT_PASS: ${{ secrets.REDSHIFT_TEST_PASS }}
+        REDSHIFT_PORT: ${{ secrets.REDSHIFT_TEST_PORT }}
+        REDSHIFT_DATABASE: ${{ secrets.REDSHIFT_TEST_DBNAME }}
+        # TODO: below value - does it need to be env-var or could we hard code it in tox and here?
```

Comment:
> below value - does it need to be env-var or could we hard code it in tox

this is just the mechanism to support GHA's `matrix` config. If we're using tox, this likely goes away